EmployeeManagementService/EmployeeManagementService.Infrastructure/Persistence/EmployeeRepository.cs

```diff
@@ -17,5 +17,24 @@ public async Task<List<Employee>> GetAllEmployees()
                 return await context.Employees.Select(e => new Employee() { FirstName  = e.FirstName, LastName = e.LastName, Role = e.Role, Username = e.Username, Active = e.Active }).ToListAsync();
             }
         }
+
+        public async Task CreateEmployee( string firstName, string lastName, string username, byte[] password, string role)

```

Comment:
Let's add an extra parameter Active. But set the default of Active to true in the parameters.

That way, methods calling this create method will not need to pass in Active as parameter if they want to create an employee with true.

At the same time, by creating the Active parameter, we give ourselves the options in the future to call this method and pass in false as input if we want to create an employee whose initial state is not active

Reply:
Done